examples/webpack/main.js

```diff
@@ -0,0 +1,31 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/licenses/publicdomain/
+
+// Hello world example for webpack.
+
+require('pdfjs-dist');
+
+var pdfPath = '../helloworld/helloworld.pdf';
+
+// It is also possible to disable workers via `PDFJS.disableWorker = true`
+
+// Loading a documents.
+var loadingTask = PDFJS.getDocument(pdfPath);
+loadingTask.promise.then(function (pdfDocument) {
+  // Getting a first page
+  return pdfDocument.getPage(1).then(function (pdfPage) {
+    // Display page on the existing canvas with 100% scale.
+    var viewport = pdfPage.getViewport(1.0);
+    var c = document.getElementById('theCanvas');
```

Comment:
Let's call this variable `canvas` (and below) as single-character variable names are less readable.